### ISSUE TITLE
[BUG] Doom does not work as expected with Emacs installed using `snap`

### ISSUE THREAD

**Issue by olofwalker** [hlissner/doom-emacs#2269]
**What did you expect to happen?**
1. Installed Emacs using `snap install emacs --classic`
2. Installed Doom
3. Running Emacs would show an emacs instance with the Doom theme and functions

**What actually happened?**
...
When launching Emacs from the Gnome menu, Emacs does not pick up Doom themes and other settings.

**Additional details:**
Ubuntu 19.10
Emacs 26.3 installed using `sudo snap install emacs --classic`
Latest Doom cloned from the repo

**Steps to reproduce:**
1. Installed Emacs using `snap install emacs --classic`
2. Installed Doom
3. Run Emacs by launching it from the Gnome 
4. Emacs should start without the expected Doom theme 

Note: When you run `doom run` everything will work as normal.

**Comment by olofwalker** [comment]
Also, Thank you for Doom Emacs, it is really helpful!

**Comment by hlissner** [comment]
I'm afraid I know nothing about `snap`. I'll have to hope this has nothing to do with the installation method!

Could you try running `doom doctor`? It will diagnose and complain about common issues with your environment that may interfere with Doom.

If it doesn't provide any clues, could you supply a screenshot of the screen you're seeing?

**Comment by colbell** [comment]
I've installed emacs from snap on Zorin OS 15.1 which is based on Ubuntu 18.4. So far it works (I just installed the OS today). My doom config gets loaded and I'm able to edit my elixir files, run magit etc.

 I didn't install from the command line, I used the GUI software tool that comes with Zorin.

$ snap info emacs 
<snip/>
snap-id:      4gE2S31SnK2LeqsZF8XTPuHOlqlzbqsW                   
tracking:     stable                                                                          
refresh-date: yesterday at 16:59 AEDT
channels:                                                                                  
  stable:    26.3            2019-12-10 (296) 120MB classic                                
  candidate: 26.3            2019-12-10 (296) 120MB classic
  beta:      27.0.60-c01f55f 2020-01-07 (320) 141MB classic
  edge:      28.0.50-4f2b415 2020-01-07 (316) 145MB classic                               
installed:   26.3                       (296) 120MB classic
